### ISSUE TITLE
Problem: airbnb rules don't work with eslint-plugin-vue

### ISSUE THREAD

**Issue by marceloavf** [vuejs/eslint-plugin-vue#259]
**Tell us about your environment**

* **ESLint Version:** 4.12.0
* **eslint-plugin-vue Version:** 3.11.0
* **Node Version:** 8.9.1

**Please show your full configuration:**

```js
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true,
  },
  extends: [
    'plugin:vue/essential',
    'airbnb-base'
  ],
  // required to lint *.vue files
  plugins: [
    'html'
  ],
......
}
```

**What did you expect to happen?**
Show warnings about `no-console` to keep track of them to clean up later

**What actually happened? Please include the actual, raw output from ESLint.**
Don't show any error of `airbnb eslint base`

**Comment by mysticatea** [comment]
Thank you for this issue.

Please remove HTML plugin:

```diff
module.exports = {
  root: true,
  parserOptions: {
    parser: 'babel-eslint',
    sourceType: 'module'
  },
  env: {
    browser: true,
  },
-  // required to lint *.vue files
-  plugins: [
-    'html'
-  ],
......
}
```

**Comment by marceloavf** [comment]
Works great,
Thank you!

**Comment by lyyourc** [comment]
if I removed the `eslint-plugin-html`, my `eslint-plugin-prettier` won't work again @mysticatea

**Comment by mysticatea** [comment]
@lyyourc This plugin requires whole source code of `.vue` file to lint `<template>`. I'm not familiar with `eslint-plugin-prettier`, but I guess `eslint-plugin-prettier` does not support HTML code itself.

**Comment by jasonxia23** [comment]
@lyyourc 

> It tries to parse the vue (html) file with the Babylon (javascript) parser which doesn't work. The vue file support has not yet landed in prettier (we will do a release on Wednesday), so until then it's not going to work.

https://github.com/prettier/prettier/issues/2097
